### PR TITLE
move dashboards file

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

approval should be trivial, there is one more step needed after this on Learn.

### CODE REVIEW

File: docs/glossary.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Use the alphabatized list below to find the answer to your single-term questions
 
 ## D
 
-- [**Dashboards**](https://github.com/netdata/netdata/blob/master/docs/cloud/visualize/dashboards.md): Out-of-the box visual presentation of metrics that allows you to make sense of your infrastructure and its health and performance.
+- [**Dashboards**](https://github.com/netdata/netdata/blob/master/docs/category-overview-pages/accessing-netdata-dashboards.md): Out-of-the box visual presentation of metrics that allows you to make sense of your infrastructure and its health and performance.
```
Comment (ilyam8) [inline docs/glossary.md:48]: Should this be **Dashboard**? Why plural?

**Review by ilyam8 — APPROVED** [review]

File: docs/glossary.md (hunk 1 above)
Comment (Ancairon) [reply docs/glossary.md:48]: yeah, I'll do it singular, nice catch

Comment (Ancairon) [inline docs/glossary.md:48]:
```suggestion
- [**Dashboard**](https://github.com/netdata/netdata/blob/master/docs/category-overview-pages/accessing-netdata-dashboards.md): Out-of-the box visual presentation of metrics that allow you to make sense of your infrastructure and its health and performance.
```

Comment (ilyam8) [reply docs/glossary.md]: > Out-of-the-box visual representation of metrics to provide insight into your infrastructure, its health and performance.

File: docs/glossary.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Use the alphabatized list below to find the answer to your single-term questions
 
 ## D
 
-- [**Dashboards**](https://github.com/netdata/netdata/blob/master/docs/cloud/visualize/dashboards.md): Out-of-the box visual presentation of metrics that allows you to make sense of your infrastructure and its health and performance.
+- [**Dashboard**](https://github.com/netdata/netdata/blob/master/docs/category-overview-pages/accessing-netdata-dashboards.md): Out-of-the box visual presentation of metrics that allow you to make sense of your infrastructure and its health and performance.
```
Comment (ilyam8) [inline docs/glossary.md:48]:
```suggestion
- [**Dashboard**](https://github.com/netdata/netdata/blob/master/docs/category-overview-pages/accessing-netdata-dashboards.md): Out-of-the-box visual representation of metrics to provide insight into your infrastructure, its health and performance.
```

I think that

> allow you to make sense of your infrastructure

sounds strange. To make sense of your infrastructure kek

**Pull request merged**